README.md

```diff
@@ -108,7 +108,56 @@ Migration guide:
 This function is deprecated. Use the `torch.nn.attention.sdpa_kernel` context manager instead.
 
 Migration guide:
+
 Each boolean input parameter (defaulting to true unless specified) of `sdp_kernel` corresponds to a `SDPBackened`. If the input parameter is true, the corresponding backend should be added to the input list of `sdpa_kernel`.
 
+### TOR102 Unsafe use of function 
+
+#### torch.load
```

Comment:
There should not be a level 3 and level 4 header, as TOR102 is just one item.
See TOR003 for example.